### ISSUE TITLE
[Suggestion] Screen mirror module 

### ISSUE THREAD

**Issue by Dijkstra1** [McJty/RFTools#700]
A module that is bound to a screen, that when is in another screen display the whats on the screen bound and pass button clicks back to first screen.

**Comment by McJty** [comment]
Do you mean so you can have multiple screens that are all configured exactly like a 'master' screen?

**Comment by Dijkstra1** [comment]
Yes but if update 'master' then all the one mirroring it would update as well
